Makefile

```diff
@@ -8,76 +8,76 @@ NAMESPACE=ods
 
 # JENKINS
 ## Install or update Jenkins agent resources.
-jenkins: jenkins-apply-build jenkins-start-build
+jenkins: apply-jenkins-build start-jenkins-build
 .PHONY: jenkins
 
 ## Update OpenShift resources related to Jenkins agent images.
-jenkins-apply-build: jenkins-apply-build-airflow jenkins-apply-build-golang jenkins-apply-build-maven jenkins-apply-build-nodejs jenkins-apply-build-python jenkins-apply-build-scala
-.PHONY: jenkins-apply-build
+apply-jenkins-build: apply-jenkins-build-airflow apply-jenkins-build-golang apply-jenkins-build-maven apply-jenkins-build-nodejs10-angular apply-jenkins-build-python apply-jenkins-build-scala
+.PHONY: apply-jenkins-build
 
 ## Update OpenShift resources related to Jenkins airflow agent image.
-jenkins-apply-build-airflow:
+apply-jenkins-build-airflow:
 	cd common/jenkins-slaves/airflow/ocp-config && tailor apply --namespace ${NAMESPACE}
-.PHONY: jenkins-apply-build-airflow
+.PHONY: apply-jenkins-build-airflow
 
 ## Update OpenShift resources related to Jenkins golang agent image.
-jenkins-apply-build-golang:
+apply-jenkins-build-golang:
 	cd common/jenkins-slaves/golang/ocp-config && tailor apply --namespace ${NAMESPACE}
-.PHONY: jenkins-apply-build-golang
+.PHONY: apply-jenkins-build-golang
 
 ## Update OpenShift resources related to Jenkins maven agent image.
-jenkins-apply-build-maven:
+apply-jenkins-build-maven:
 	cd common/jenkins-slaves/maven/ocp-config && tailor apply --namespace ${NAMESPACE}
-.PHONY: jenkins-apply-build-maven
+.PHONY: apply-jenkins-build-maven
 
 ## Update OpenShift resources related to Jenkins nodejs agent image.
-jenkins-apply-build-nodejs:
+apply-jenkins-build-nodejs10-angular:
```

Comment:
don't we miss nodejs12? or not required yet?

Reply:
Well - I wasn't sure. It is not referenced anywhere so I decided to leave that like it is for now.